Extension/package.nls.json

```diff
@@ -26,6 +25,8 @@
     "c_cpp.command.RunCodeAnalysisOnOpenFiles.title": "Run Code Analysis on Open Files",
     "c_cpp.command.RunCodeAnalysisOnAllFiles.title": "Run Code Analysis on All Files",
     "c_cpp.command.ClearCodeAnalysisSquiggles.title": "Clear Code Analysis Squiggles",
+    "c_cpp.command.BuildAndDebugFile.title": "Debug Active File",
```

Comment:
We should add "C/C++" in the button titles in case customers have the Java, Python, or Code Runner extensions installed. I also don't think we need "Active" in the title if we have "C/C++" since the buttons will only be shown when a source file is active.

Let's use "Debug C/C++ File" and "Run C/C++ File"

Reply:
This would be consistent with Python:
![image](https://user-images.githubusercontent.com/44306969/152419918-70060ee2-7fb8-4a10-a82d-5d42aa6a80c8.png)

Reply:
good idea